### ISSUE TITLE
AWS Glacier ?

### ISSUE THREAD

**Issue by aramonc** [thephpleague/flysystem#329]
Is anyone working on a [Glacier](http://aws.amazon.com/documentation/glacier/) adapter? 

Let me rephrase: If no one is working on this already, my company is considering using Glacier and we would like to have a single API for all of the filesystems. We would have no trouble implementing this. Just don't want to double up effort.

**Comment by frankdejonge** [comment]
Closing due to lack of interest.

**Comment by aramonc** [comment]
> @aramonc, I don't suppose ya'll ended up implementing one?

@seth-shaw-unlv no, I think we gave up on the idea of archiving the files. I haven't been with the company I mentioned in the OP for a few years so it's possible they did implement something after I left.
